### PR TITLE
Add header section

### PR DESCRIPTION
**Pull request by srikant96**

### CODE REVIEW

File: blogs/01/index.html [hunk 1]
```diff
@@ -1,4 +1,25 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/common.css">
+    <link rel="stylesheet" href="styles/header.css">
+    <title>Mini Blog</title>
+</head>
+<body>
+        <nav class="header">
```
Comment (amezng) [inline blogs/01/index.html:11]: Use header element to wrap the nav element

File: blogs/01/styles/header.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    color: #2f89fc;
+}
+nav.header>ul>li>a>img{
+    height:20px;
+}
+
+@media screen and (max-width:425px){
+	nav.header>ul{
+		display:none;
+}
```
Comment (amezng) [inline blogs/01/styles/header.css:39]: Mobile view is incomplete. Complete the css for responsive design.

**Review by amezng — CHANGES_REQUESTED** [review]
MTC

File: blogs/01/styles/header.css (hunk 2 above)
Comment (srikant96) [reply blogs/01/styles/header.css:39]: I used display none to remove the Ul element, in blog website only Mini blog is there is left side when  we switch  mobile view  . So  i just removed  removed ul section.

@media screen and (max-width:425px){
	nav.header>ul{
		display:none;
}
}

File: blogs/01/index.html (hunk 1 above)
Comment (srikant96) [reply blogs/01/index.html:12]: Done

File: blogs/01/styles/header.css (hunk 2 above)
Comment (amezng) [reply blogs/01/styles/header.css:39]: What about the hamburger icon on the right side?

Comment (srikant96) [reply blogs/01/styles/header.css:57]: done

**Pull request merged**